src/app/component/MissionGroup/MissionGroup.jsx

```diff
@@ -53,20 +54,15 @@ const MissionGroup = ({ callToAction, group, groupCallToAction, showViewRoute })
       onClick(mission.uid);
     });
   };
-  const onClickViewRouteButton = () => {
-    console.log("View Routes");
-  };
 
   const viewRoute = showViewRoute && (
-    <Button
-      fullWidth={true}
-      variant="text"
-      startIcon={<MuiMapIcon />}
-      onClick={() => onClickViewRouteButton()}
-    >
-      View Route
-    </Button>
+    <ShowDeliveryRoute
+      missions={missions}
+      isEmpty={isEmpty(missions)}
+      isLoaded={isLoaded(missions)}
```

Comment:
Done. I did it this way because some*&^ thing wasn't working and I was paranoid about the magic state fairies, but in the end it was fine to just do this as you suggest.